### ISSUE TITLE
Cognitive Services API missing

### ISSUE THREAD

**Issue by kpfaulkner** [Azure/azure-rest-api-specs#731]
Are the Azure Cognitive Services APIs going to be added to this repo?
I see on the API sites (eg https://dev.projectoxford.ai/docs/services/56f91f2d778daf23d8ec6739/operations/56f91f2e778daf14a499e1fa ) that a Swagger definition is available.

If I were to contribute, would I simply need to gather these misc swagger files and add them to a forked repo and then PR? Or is this repo a lot more than that?

Thanks

Ken

**Comment by mcardosos** [comment]
There already is a swagger for [cognitive services](https://github.com/Azure/azure-rest-api-specs/blob/master/arm-cognitiveservices/2016-02-01-preview/swagger/cognitiveservices.json), but it look quite different than the one @kpfaulkner is linking. Maybe someone in Cognitive Services team knows a bit more?
cc @daniel-ms

**Comment by kpfaulkner** [comment]
Hi

I thought the existing swagger was for management of the Cognitive services as opposed to actually "using" them (so to speak).

Actually nearly every swagger definition I've looked at in this repo seems to be about managing as opposed to using, or maybe I'm just looking at the wrong ones?

**Comment by mcardosos** [comment]
No, you are right. Most of this swaggers are for managing. The ones for using are the ones outside an `arm` folder.

**Comment by kpfaulkner** [comment]
In that case, doesn't that explain why the swagger is different? The existing one is for managing but from what I see each section of CSvcs (Face, Vision, Speech etc) all have their own Swagger definitions for the actual using of the APIs and not just managing?

**Comment by mcardosos** [comment]
Then my question for @daniel-ms would be more on the side of why that swagger doesn't live in this repo as a dataplane swagger.

**Comment by salameer** [comment]
Pinging @daniel-ms

**Comment by felixwa** [comment]
This swagger file is for managing Cognitive Services "accounts", that is, create/update/delete accounts. For using each individual Cognitive Services API (like Face, Vision etc.), you may want to use their own SDK in GitHub. And so far as I know they don't have swagger files.
Hope this clarifies the situation and can we get this issue closed?

**Comment by mcardosos** [comment]
@felixwa 
The checked-in swagger if for managing accounts, yes. About the individual APIs there is a [swagger available](https://dev.projectoxford.ai/docs/services/56f91f2d778daf23d8ec6739/export?DocumentFormat=Swagger&ApiName=Computer%20Vision%20API%20-%20v1.0) in the link @kpfaulkner provided (there is an API definition button in the corner that includes the swagger).
We were wondering if there is a plan to merge that swagger here in this repo.

**Comment by felixwa** [comment]
@mcardosos , I see but no, we don't have plan to merge them here. In fact those swagger files are **automatically generated** by Azure API Management, not maintain by us. I don't think it provides much value we put them into this repo.
